openapi/openai.chat/Ballerina.toml

```
@@ -6,7 +6,7 @@ name = "openai.chat"
 icon = "icon.png"
 distribution = "2201.4.1"
 repository = "https://github.com/ballerina-platform/openapi-connectors/tree/main/openapi/openai.chat"
-version = "1.0.4"
+version = "1.0.5"
```

As there are some breaking changes, we need to bump minor version
```suggestion
version = "1.1.0"
```

---

Addressed in [ff8ee76](https://github.com/ballerina-platform/openapi-connectors/pull/800/commits/ff8ee76c6d1c6f2cfa2ec0ed89bd69088e036581)